lms/djangoapps/course_goals/models.py

```diff
@@ -35,8 +36,15 @@ class Meta:
     course_key = CourseKeyField(max_length=255, db_index=True)
     # The goal a user has set for the number of days they want to learn per week
     days_per_week = models.PositiveIntegerField(default=0)
+
     # Controls whether a user will receive emails reminding them to stay on track with their learning goal
     subscribed_to_reminders = models.BooleanField(default=False)
+
+    # With this token, anyone can unsubscribe this user from reminders. That's a mild enough action that we don't worry
+    # about not keeping this key around long term in the database or using a higher-security generator than uuid4.
```

Comment:
This comment is reading a little funny to me. You say "we don't worry about not keeping this key around long term in the database" while explicitly saving the key in the database.

Reply:
Ah sorry, I meant "we are not worried about the risk of keeping this key around long term" not that "we are not keeping this key around long term".  Will clarify text, good catch.

Reply:
Updated text - hopefully clearer? And added a test line for the `course_title` response.